src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int INDEX_ACTION = 0;
+    private static final int INDEX_AMOUNT = 1;
+    private int buy = 0;
+    private int supply = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        List<String> listFromFile;
+        String[] dividedString;
+
+        try {
+            listFromFile = Files.readAllLines(fromFile.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read fromFile", e);
+        }
+        for (String s : listFromFile) {
+            dividedString = s.split(",");
+            if (dividedString[INDEX_ACTION].equals("buy")) {
+                buy += Integer.parseInt(dividedString[INDEX_AMOUNT]);
+            } else {
+                supply += Integer.parseInt(dividedString[INDEX_AMOUNT]);
+            }
+        }
+        writeToFile(toFileName);
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = createREport(dataFromFile);
        writeToFile(toFileName, report);
```
this is how your code should be divided